p24/src/main.rs

```diff
@@ -0,0 +1,27 @@
+pub fn f3(s: &mut [i32], length: usize) -> &i32 {
+    println!("===========================");
+    println!("{:?}", s);
+    return &mut s[s.len() - length];
+}
+
+fn main() {
```

Comment:
After you finished checking your code, do not forget to remove `main.rs`, since the problem description asks to implement a library crate.